commands/src/main/java/org/jclouds/karaf/commands/table/internal/ScriptEngineShellTable.java

```diff
@@ -51,9 +47,7 @@ public String evaluate(Object obj, String expression) {
     try {
       scriptEngine.put(getType(), obj);
       result = String.valueOf(scriptEngine.eval(expression));
-    } catch (Exception ex) {
-      //Ignore
-    }
+    } catch (Exception ignored) {}
     return result;
```

Comment:
> That would make more sense for users than a bunch of empty lines.

I find it difficult to decide one way or the other without knowing what's failing here, and whether that's more of the "fatal error" category, or whether it could be the result of user error and be recoverable. It would certainly be good to set `result` for user-related errors, and perhaps to blow up for errors that indicate a broken configuration.

I'm just not sure if we can pick those apart simply from the exception type?